lib/batch_verification/src/client/metrics.rs

```diff
@@ -1,11 +1,48 @@
-use vise::{Gauge, Metrics};
+use vise::{Counter, Gauge, Metrics};
 
 #[derive(Debug, Metrics)]
 #[metrics(prefix = "batch_verification_client")]
 pub struct BatchVerificationClientMetrics {
-    pub block_cache_size: Gauge<usize>,
+    /// Number of blocks currently cached
+    block_cache_size: Gauge<usize>,
+    /// Lowest block height in the cache
+    block_cache_from_height: Gauge<u64>,
```

Comment:
nit: while valid, we never use term"block_height" we always say "block_number". Please change for consistency

Reply:
Will do in followup.